lib/github.ts

```diff
@@ -11,17 +11,31 @@ type Options = {
   branch: string
 }
 
+const checkBranches = async (octokit: any, branch: string): Promise<string> => {
```

Comment:
If I read your code correctly, this function basically checks if `master` should be used instead of `develop` right?

Reply:
yes